### PR TITLE
Refactor views

### PR DESCRIPTION
**Pull request by aaronkrogers**
Refactored the views.py file to replace the old try/except clauses with simpler if statements.
This does nothing to original functionality and is mostly intended to create working and more readable code to aid in extending views in the future

### CODE REVIEW

File: users/views.py [hunk 1]
```diff
@@ -29,65 +29,45 @@ def badge(request, person_id):
...
+        'person': person,
+        'error_message': None,
+    }
+
+    # get values from the page form
+    badge_name = request.POST.get('name', '')
+    badge_presenter = request.POST.get('presenter', '')
+
+    # verify that form fields have be filled
+    if '' in (badge_name, badge_presenter):
```
Comment (mtik00) [inline users/views.py]: This is _logically_ fine, but I think it's more Pythonic to write this instead:
```suggestion
    if not (badge_name or badge_presenter):
```

**Review by mtik00 — CHANGES_REQUESTED** [review]
Looking good!  Just one small change requested.

File: users/views.py (hunk 1 above)
Comment (aaronkrogers) [reply users/views.py]: Agreed, I sometimes forget about the Truthiness of empty data structures

**Review by mtik00 — APPROVED** [review]
Thanks

**Pull request merged**